pkg/operator/operator.go

```diff
@@ -11,6 +11,12 @@ import (
 	"github.com/golang/glog"
 	configclientset "github.com/openshift/client-go/config/clientset/versioned"
 	installertypes "github.com/openshift/installer/pkg/types"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
```

Comment:
the ideal case for import grouping that i would like is:
```
<stdlib>

<external>

<k8s>

<internal>
```

Reply:
> the ideal case for import grouping...

Why split out k8s?  Alphabetizing non-stdlib packages will get them sorted together anyway, and `gofmt` will (I think?) enforce that ordering witin a block of contiguous imports.

Reply:
Made the change but lmk if you want to use a diff import format as @wking suggested.